src/EdFi.AnalyticsMiddleTier.DataStandard2/Qews/MSSQL/0003-View-StudentEnrolledSectionGradeTrend-Create.sql

```diff
@@ -12,7 +12,7 @@ CREATE VIEW [analytics].[qews_StudentEnrolledSectionGradeTrend] AS
 			[StudentSectionDim].[SchoolKey],
 			CASE WHEN [StudentSectionDim].[Subject] = 'Mathematics' THEN 'Math' 
 				 WHEN [StudentSectionDim].[Subject] IN ('English Language Arts', 'Reading', 'Writing') THEN 'English' END as [Subject],
-			CONCAT([DateDim].[CalendarYear], '-', FORMAT([DateDim].[Month], '00')) as [Month],
+			CONCAT([DateDim].[CalendarYear], '-', RIGHT(CONCAT('00', [DateDim].[Month]),2)) as [Month],
```

Comment:
It looks like you made these changes and then created new scripts. Since the right way is to create new scripts (and if you don't mind) I will undo these changes.

Reply:
These views are using DateDim and we changed DateDim datatypes. Then, the old scripts won't run. To handle that I added the new version (to update the view in an update) but also the old scripts. If we don't change that, the operational testcases fail.